handling-forms/controlled-forms.md

```diff
@@ -226,50 +226,61 @@ Sofia's app currently has the following components:
 ```js
 import { useState } from 'react';
 import StudentList from './components/StudentList';
+import ClassInfo from './components/ClassInfo';
 
 function App() {
-    const [studentData, setStudentData] = useState([
-        {
-            id: 1,
-            nameData: 'Ada',
-            emailData: 'ada@dev.org',
-            isPresentData: false
-        },
-        {
-            id: 2,
-            nameData: 'Soo-ah',
-            emailData: 'sooah@dev.org',
-            isPresentData: false
-        },
-        {
-            id: 3,
-            nameData: 'Chrissy',
-            emailData: 'chrissy@dev.org',
-            isPresentData: true
-        }
-    ]);
-
-    const updateStudentData = updatedStudent => {
-        const students = studentData.map(student => {
-            if (student.id === updatedStudent.id) {
-                return updatedStudent;
-            } else {
-                return student;
-            }
-        });
-
-        setStudentData(students);
-    };
+  const [studentData, setStudentData] = useState([
+    {
+      id: 1,
+      nameData: 'Ada',
+      emailData: 'ada@dev.org',
+      isPresentData: false,
+    },
+    {
+      id: 2,
+      nameData: 'Soo-ah',
+      emailData: 'sooah@dev.org',
+      isPresentData: false,
+    },
+    {
+      id: 3,
+      nameData: 'Chrissy',
+      emailData: 'chrissy@dev.org',
+      isPresentData: true,
+    },
+  ]);
+
+  const toggleStudentPresence = (studentId) => {
+    // calculate the updated student data by finding the student that matches
```

Comment:
I've been removing all the comments when re-presenting the state-of-the-app code.